src/Network/Algorand/Client.hs

```diff
@@ -9,30 +9,22 @@ module Network.Algorand.Client
 
   , AlgoIndexer (..)
   , connectToIndexer
-
-  , module Api
   ) where
 
 import qualified Data.Text as T
 
 import Control.Exception.Safe (MonadThrow, throwM)
 import Control.Monad.IO.Class (MonadIO, liftIO)
 import Network.HTTP.Client.TLS (newTlsManager)
-import Servant.API.Generic (fromServant)
-import Servant.Client (ClientEnv, mkClientEnv, parseBaseUrl, runClientM)
+import Servant.Client (mkClientEnv, parseBaseUrl, runClientM)
 import Servant.Client.Generic (AsClientT, genericClientHoist)
 
-import qualified Network.Algorand.Api as Api
-
-import Network.Algorand.Api (Api (..), ApiAny (..), ApiIdx2, ApiV2, Version (..))
-import Network.Algorand.Definitions (Host, Network, NetworkError (WrongNetwork))
-
-apiClient :: forall m' . MonadIO m' => ClientEnv -> Api (AsClientT m')
```

Comment:
Why did you remove `apiClient` helper? Now I see `genericClientHoist $ \x -> liftIO $ runClientM x env >>= either throwM pure` in 2 places.

Reply:
because you asked to delete `API` type which combine node and indexer api's) 
I thought about type signature of `apiClient` and didn't found solution without `API` type. Any ideas?

Reply:
I've made it)